### PR TITLE
DEVELOPER-3682 and 	DEVELOPER-3623 Fixed Topics' white cards

### PR DESCRIPTION
**Pull request by adelasofia**
HACKATHON

**1.  [DEVELOPER-3682 - Topic white cards are not fully loaded for mobile views](https://issues.jboss.org/browse/DEVELOPER-3682)**

What to check: 16 white cards should be loaded on desktop, tablet and mobile.

Also fixed width of 4 top white cards on tablet.

**2. [DEVELOPER-3623 - Topics page cards overflow and disrupt page structure](https://issues.jboss.org/browse/DEVELOPER-3623)**

In short, this was fixed (screenshot from /devops) : https://www.dropbox.com/s/81kihngprckaviq/Screenshot%202017-06-15%2011.22.54.png?dl=0

### CODE REVIEW

**Comment by robpblake** [comment]
retest this please

**Comment by adelasofia** [comment]
retest this please
